### DISCUSSION TITLE
index.html vs main.html

### DISCUSSION THREAD

**Discussion by vicente999** [IIC2513/Syllabus#13] [Tarea 1]
Hola! Se nos pide tener 3 archivos .html pero no me queda claro la diferencia entre el index.html y el archivo de la página principal.
Feliz si me pueden dar un ejemplo con una pagina web real.

Gracias!

**Comment by Dafnemami** [comment]
Me sumo a la pregunta! Es acaso este segundo html uno que se debe importar al index.html? No me queda claro que hacer aún.

**Comment by valeeeriquelme** [comment]
index.html es un landing page. La idea es que sea un punto de entrada a su página. Acá no tiene que estar el contenido principal (fichas), aquí se da la bienvenida al sitio. 

landing page -> https://www.codecademy.com/
about -> https://www.codecademy.com/about
pagina principal -> https://www.codecademy.com/catalog
